package.json

```diff
@@ -2,7 +2,11 @@
   "name": "compassql",
   "version": "0.20.1",
   "description": "CompassQL visualization query language",
-  "main": "build/src/cql.js",
+  "main": "build/compassql.js",
+  "unpkg": "build/compassql.min.js",
+  "jsdelivr": "build/compassql.min.js",
+  "module": "build/src/index",
```

Comment:
we don't have index.ts -- we still use the old school cql.ts as the main file.  Same for types.

Reply:
Let's correct this to cql.